deck/views.py

```diff
@@ -47,7 +51,10 @@ def get_context_data(self, **kwargs):
         return context
 
 
-class CreateEvent(BaseEventView, CreateView, FormValidRedirectMixing):
+class CreateEvent(LoginRequiredMixin, 
```

Comment:
After the `,` there should not have any blank spaces, this is happen on many places.